src/helpers/utils.ts

```diff
@@ -24,6 +25,14 @@ export function visualizationIsBetaWarning() {
     );
 }
 
+export function percentFormatter(value: number): string {
+    if (isNil(value)) {
+        return "";
+    }
+    const val = parseFloat(value.toFixed(2));
+    return `${val}%`;
```

Comment:
I recommend the shorter way, but its up to you :)
```
return isNil(value) ? "" : `${parseFloat(value.toFixed(2))}%`; 
```

Reply:
fixed